### PR TITLE
Add section references on how padding is used in QUIC

### LINKED ISSUE #194: Padding explanation needs to mention DPLPMTUD
“Padding is used by QUIC to ensure” - please add a sentence that DPLPMTUD is another reason why you might expect QUIC Padding of specific packets after connection establishment, and refer to section in QUIC transport - or people may have a surprise when they suddenly see big probe packets in their logs:-)

### PR DESCRIPTION
**Pull request by mirjak**
fixes #194

### CODE REVIEW

**Review by gorryfair — APPROVED** [review]
Looks good

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -437,14 +437,14 @@ wait before bundle frames into a packet.
...
-
-Padding is used by QUIC to ensure that the path is capable of transferring
-datagrams of at least a certain size, both during the handshake and for
-connection migration. Padding can also be used by an application to reduce
-leakage of information about the data that is sent. A QUIC implementation can
-expose an interface that allows an application layer to specify how to apply
-padding.
+arbitrarily increase the size of packets. Padding is used by QUIC to ensure
+that the path is capable of transferring datagrams of at least a certain size,
+during the handshake (see section 8.1 and 14.1 of {{!QUIC}}) and for path
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:442]:
```suggestion
during the handshake (see Sections 8.1 and 14.1 of {{!QUIC}}) and for path
```

File: draft-ietf-quic-applicability.md [hunk 2]
```diff
@@ -437,14 +437,14 @@ wait before bundle frames into a packet.
...
+validation after connection migration (see 8.2. of {{!QUIC}}) as well as for
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:443]:
```suggestion
validation after connection migration (see {{Section 8.2. of QUIC}}) as well as for
```

Not entirely sure that the reference format works, but try it.

File: draft-ietf-quic-applicability.md [hunk 3]
```diff
@@ -437,14 +437,14 @@ wait before bundle frames into a packet.
...
+Datagram Packetization Layer PMTU Discovery (DPLMTUD) (see section 14.3 of
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:444]:
```suggestion
Datagram Packetization Layer PMTU Discovery (DPLMTUD) (see Section 14.3 of
```

File: draft-ietf-quic-applicability.md [hunk 4]
```diff
@@ -437,14 +437,14 @@ wait before bundle frames into a packet.
...
+{{!QUIC}}). Padding can also be used by an application to reduce leakage of
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:445]: I would split here:

```suggestion
{{!QUIC}}).

Padding can also be used by an application to reduce leakage of
```

**Review by martinthomson — APPROVED** [review]

**Pull request merged**